packages/http/src/mocker/generator/JSONSchema.ts

```diff
@@ -20,7 +22,13 @@ jsf.option({
 });
 
 export function generate(source: JSONSchema): Either<Error, unknown> {
-  return tryCatch(() => jsf.generate(cloneDeep(source)), toError);
+  return pipe(
+    stripWriteOnly(source),
+    O.fold(
+      () => right(undefined),
+      schema => tryCatch(() => jsf.generate(schema), toError)
+    )
```

Comment:
We never fold, we rather propagate the error and handle it on the edges. This should be chained